packages/@expo/cli/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 
 ### 💡 Others
 
+- Update the README file. ([#18663](https://github.com/expo/expo/pull/18663) by [@EvanBacon](https://github.com/EvanBacon))
```

Comment:
IMO we don't need changelog entry for this change.

Reply:
Changelog is used to cherry-pick commits on the release branch, this will effect the NPM page presence.